test-data/unit/cmdline-based.test

```diff
@@ -0,0 +1,30 @@
+[case testIgnoreMissingPyTyped]
+# cmd: mypy main.py
+[file main.py]
+from a import x
+from b import y
+[file mypy.ini]
+\[mypy]
+\[mypy-a]
+ignore_missing_py_typed = true
+[file pypath/a/__init__.py]
+x = 0
+[file pypath/b/__init__.py]
+y = 0
+[out]
+main.py:2:1: error: Skipping analyzing "b": module is installed, but missing library stubs or py.typed marker  [import]
+main.py:2:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
+
+
+[case testIgnoreMissingPyTypedDontClobberStubs]
+# cmd: mypy main.py
+[file main.py]
+from a import x
+reveal_type(x)
+[file pypath/a/__init__.py]
+x = 0
+[file pypath/a-stubs/__init__.pyi]
+x: str
+[out]
+main.py:2:13: note: Revealed type is "str"
+== Return code: 0
```

Comment:
Add a test for when there's a `py.typed` file?

Reply:
added